Source/Engine/PackageBuilder/SyntaxParser.cs

```diff
@@ -852,7 +900,12 @@ private Syntax ParsePatternReference(string patternName)
                         throw SyntaxError(TextResource.AttributesAreNotAllowedForStandardPattern, patternName);
                 }
                 else
-                    result = pattern.Body;
+                    result = pattern.Body switch
+                    {
+                        TokenSyntax token => new TokenSyntax(token.TokenKind, token.Text, token.IsCaseSensitive, token.TextIsPrefix, token.TokenAttributes),
+                        VariationSyntax variation => new VariationSyntax(variation.Elements, false),
+                        _ => throw SyntaxError(TextResource.UnknownStandardPattern, pattern)
```

Comment:
I suggest to use TextResource.InternalCompilerError message, since standard patterns are managed by Nevod source code and not by a user. So if this line is encountered it means that there is a bug in Nevod itself.